handlers.go

```diff
@@ -38,13 +40,16 @@ func (sf *Sunfish) AddFile(w http.ResponseWriter, r *http.Request) {
 		w.Header().Set("Content-Type", "application/json; charset=UTF-8")
 		w.WriteHeader(422) // unprocessable entity
 		if err := json.NewEncoder(w).Encode(err); err != nil {
-			panic(err)
+			sf.logger.Println("ERROR: Could not encode json response.")
+			return
 		}
+		sf.logger.Println("INFO: Could not process the uploaded Siafile.")
```

Comment:
Cluttering the logs is a non issue in my opinion. Logging something like this let's us know what's being hit. Logs are grep-able so info's can be ignored. And we want to know if people are abusing our service or using it wrong.